src/main/java/io/nitric/api/document/Query.java

```diff
@@ -81,7 +78,7 @@
  *  import io.nitric.api.document.QueryResults;
  *  import com.example.entity.Customer;
  *
- *  QueryResults results = Documents.collection("customers")
+ *  QueryResults&lt;Customer&gt; results = Documents.collection("customers")
```

Comment:
Not change required but worth discussing later: Using markdown for code examples eliminates the need to remember to use character encoding, making it easier to read/maintain.

Reply:
Agreed